hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -256,6 +256,117 @@ def hpat_pandas_series_values_impl(self):
     return hpat_pandas_series_values_impl
 
 
+@overload_method(SeriesType, 'value_counts')
+def hpat_pandas_series_value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
+    """
+    Pandas Series method :meth:`pandas.Series.value_counts` implementation.
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_value_counts*
+
+    Parameters
+    -----------
+    normalize: :obj: boolean, default False
+        If True then the object returned will contain the relative frequencies of the unique values
+    sort: :obj: boolean, default True
+        Sort by frequencies
+    ascending: :obj: boolean, default False
+        Sort in ascending order
+    bins: :obj: integer, default None
+        *unsupported*
+    dropna: :obj: boolean, default True
+        Skip counts of NaN
+        *unsupported* for String
```

Comment:
As I understand we need to describe type of the parameter as :obj:\`boolean\`. Could you make such change for all parameters? Also please add description for `self` and unsupported parameters.